src/js/script.js

```diff
@@ -266,9 +300,33 @@ const select = {
       const event = new Event ('updated');
       thisWidget.element.dispatchEvent(event);
     }
-
   }
   
+  class Cart {
+    constructor(element){
+      const thisCart = this;
+      thisCart.products = [];
+      thisCart.getElements(element);
+      thisCart.initActions();
+      console.log('new Cart', thisCart);
+    }
+
+    getElements(element){
+      const thisCart = this;
+      thisCart.dom = {};
+      thisCart.dom.wrapper = element;
+      thisCart.dom.toggleTrigger = thisCart.element.querySelector(select.cart.toggleTrigger);
```

Comment:
Hej!
Powodem, dla którego koszyk się nie rozwija, jest fakt, że próbujesz wykonać `querySelector` na elemencie, który jest `undefined`. Zauważ, że w konsoli wyświetlany jest błąd.
Jeśli się przyjrzysz, do zorientujesz się, że w obiekcie `thisCart` nie ma właściwości `element`, dlatego właśnie jej wartość to `undefined`. Możesz złapać `element` odwołując się bezpośrednio do elementu przekazanego w argumencie constructora:
```js
thisCart.dom.toggleTrigger = element.querySelector(select.cart.toggleTrigger);
``` 
lub - nawet lepiej - poprzez odwołanie się do właściwości `dom.wrapper` obiektu `cart`:
```js
thisCart.dom.toggleTrigger = thisCart.dom.wrapper.querySelector(select.cart.toggleTrigger);
```